### PR TITLE
feat: create_compute_hyperdisk

### PR DESCRIPTION
**Pull request by gryczj**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed guidelines from [CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md) and [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [x] **Tests** pass:   `npm test` (see [Testing](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#run-the-tests-for-a-single-sample))
- [x] **Lint** pass:   `npm run lint` (see [Style](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#style))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] This pull request is from a branch created directly off of `GoogleCloudPlatform/nodejs-docs-samples`. Not a fork.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new sample directory, and I created [GitHub Actions workflow](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#adding-new-samples) for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: compute/test/createInstanceTemplates.test.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const compute = require('@google-cloud/compute');
 
 const {describe, it} = require('mocha');
 const cp = require('child_process');
-const {assert} = require('chai');
```
Comment (gryczj) [inline compute/test/createInstanceTemplates.test.js:21]: chai module causes problem when running the tests on github and I changed the import to the same as used in batch module

**Comment by gryczj** [comment]
@subfuzion Could you help me with these problems? https://github.com/GoogleCloudPlatform/nodejs-docs-samples/actions/runs/10387680924/job/28761687813?pr=3795. Many tests- not really related to my changes are failing and I'm not sure what can cause it.

**Comment by subfuzion** [comment]
> @subfuzion Could you help me with these problems? https://github.com/GoogleCloudPlatform/nodejs-docs-samples/actions/runs/10387680924/job/28761687813?pr=3795. Many tests- not really related to my changes are failing and I do not know what can cause it.

@gryczj Yes, I should be able to help you out; hang in there a bit longer, I'm clearing out the backlog and should be able to get to yours today as well.

File: compute/test/createInstanceTemplates.test.js [hunk 2]
```diff
@@ -34,7 +34,8 @@ const createInstance = async (projectId, zone, instanceName) => {
         {
           initializeParams: {
             diskSizeGb: '250',
-            sourceImage: 'projects/debian-cloud/global/images/family/debian-10',
+            sourceImage:
```
Comment (gryczj) [inline compute/test/createInstanceTemplates.test.js:37]: used the same path as in https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/compute/create-instance-templates/createTemplateFromInstance.js

**Comment by gryczj** [comment]
Looks like the source projects/debian-cloud/global/images/family/debian-10 does not exist. I changed the path to use **debian-11** and that solved the problem. I can move these changes to separate PR and mark as **refactor** if there is a need.

**Pull request merged**